include/glow/Graph/Nodes.h

```diff
@@ -119,12 +119,22 @@ class Variable final : public Node {
 using VariableNode = Variable;
 
 /// Calculate the size of the output tensor based on the convolution parameters.
-inline std::pair<size_t, size_t> calculateConvOutputDims(size_t sx, size_t sy,
+inline std::pair<size_t, size_t>
+calculateConvOutputDims(size_t sx, size_t sy, size_t filterSize, size_t stride,
+                        llvm::ArrayRef<size_t> pads) {
+  // pads {pad_top, pad_left, pad_bottom, pad_right}
+  size_t outsx = ((sx + pads[0] + pads[2] - filterSize) / stride + 1);
```

Comment:
Maybe add an enum for top/left/bottom/right so it's clear which is being accessed? It's a bit hard to read/error prone to always use the numeric indices.